src/styles/index.scss

```diff
@@ -1,3 +1,164 @@
 body {
   margin: 0;
 }
+
+* {
+  box-sizing: border-box;
+
+  font-family: Roboto, sans-serif;
+}
+
+:root {
+  --main-accent: #060B35;
+  --secondary: #616070;
+  --blue-accent: #00ACDC;
+  --white: #FFF;
+}
+
+.card {
+  margin: 0;
+  padding: 0;
+  width: 200px;
+
+  flex-shrink: 0;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background-color: #fff;
+}
+
+.card__image{
+  border: 0;
+  width: 160px;
+  height: 134px;
+  flex-shrink: 0;
+  padding: 0;
+  margin: 32px 19px 40px;
+}
+
+.card__image_product {
+  border: 0;
+  width: 160px;
+  height: 134px;
+}
+
+.card__text {
+  color: var(--main-accent);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 0 16px;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  margin: 0 16px;
+  margin-right: 68px;
+
+  color: var(--secondary);
+  font-size: 10px;
+  font-weight: inherit;
+  line-height: 14px;
+}
+
+.card__container_review {
+  border: 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 16px 16px 0;
+}
+
+.card__container_review_stars {
```

Comment:
we should have two blocks: .card, .stars. Therefore, we must two files as Semen wrote to you